### PR TITLE
Deprecated CLI command deletion and rpm update

### PR DESCRIPTION
**Pull request by Preeticp**
- This PR deletes redundant commands and modifies another one.
- It also updates the RPM and CLI links for installation
- It has been reviewed by SME and QE
- It needs to be merged to 4.5 and 4.6

### CODE REVIEW

**Comment by Preeticp** [comment]
@ppitonak request you to review, please.

File: modules/op-tkn-pipeline-management.adoc
Comment (ppitonak) [inline modules/op-tkn-pipeline-management.adoc:14]: All three download links in this doc need to be updated to https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/0.11.0/

**Review by ppitonak — CHANGES_REQUESTED** [review]
`tkn resource create` still exists but only in interactive mode, the `-f` flag was removed, our example needs to be updated

**Comment by piyush-garg** [comment]
Following example in `tkn resource create` command need to be modified/removed. We have the command there by `-f` flag has been removed

```
Example: Create Pipeline Resource defined by the myresource.yaml file in a namespace
```

File: modules/op-tkn-pipeline-resource-management.adoc
Comment (piyush-garg) [inline modules/op-tkn-pipeline-resource-management.adoc:23]: This command will be `tkn resource create -n myspace`

**Comment by Preeticp** [comment]
@ppitonak @piyush-garg fixed, please LGTM if you see fit.

**Review by ppitonak — APPROVED** [review]
LGTM

File: modules/op-tkn-pipeline-resource-management.adoc
Comment (bergerhoffer) [inline modules/op-tkn-pipeline-resource-management.adoc:25]: that will ask your inputs on -> that asks for input on

**Review by bergerhoffer — COMMENTED** [review]
Just one suggestion.

Then be sure to squash your commits please.

**Comment by Preeticp** [comment]
@bergerhoffer fixed the comment and squashed the commits. Please merge. This applies to 4.5 and 4.6.

**Comment by bergerhoffer** [comment]
LGTM, just need to remove WIP before I can merge.

**Pull request merged**
